src/main/java/org/creekservice/kafka/test/perf/implementations/DevHarrelImplementation.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2023 Creek Contributors (https://github.com/creek-service)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.creekservice.kafka.test.perf.implementations;
+
+import static org.creekservice.kafka.test.perf.testsuite.SchemaSpec.DRAFT_2019_09;
+import static org.creekservice.kafka.test.perf.testsuite.SchemaSpec.DRAFT_2020_12;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import dev.harrel.jsonschema.Dialects;
+import dev.harrel.jsonschema.SchemaResolver;
+import dev.harrel.jsonschema.SpecificationVersion;
+import dev.harrel.jsonschema.Validator;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Set;
+import org.creekservice.kafka.test.perf.TestSchemas;
+import org.creekservice.kafka.test.perf.model.TestModel;
+import org.creekservice.kafka.test.perf.testsuite.AdditionalSchemas;
+import org.creekservice.kafka.test.perf.testsuite.SchemaSpec;
+
+@SuppressWarnings("FieldMayBeFinal") // not final to avoid folding.
+public class DevHarrelImplementation implements Implementation {
+
+    private static final MetaData METADATA =
+            new MetaData(
+                    "json-schema (dev.harrel)",
+                    "json-schema (dev.harrel)",
+                    Language.Java,
+                    Licence.MIT,
+                    Set.of(DRAFT_2020_12, DRAFT_2019_09),
+                    "https://github.com/harrel56/json-schema");
```

Comment:
Hey @harrel56,

I'm about to merge another PR: https://github.com/creek-service/json-schema-validation-comparison/pull/54

This should NOT stop your code compiling (I hope).  However, it would be great if you could update your PR as I've noted below to set a color for your impl in the result charts.

,

```suggestion
                    "https://github.com/harrel56/json-schema",
                     new Color(235, 54, 172));
```

Reply:
Changed the color to something else - felt like the pinky one was already taken? Honestly hard to tell how it's going to look like on the charts. BTW this chart page look really nice

Reply:
You can build them locally, but I've not yet documented how to.  Feel free to change the color in a new PR if you don't like it.